module/data/item/fields/summons-field.mjs

```diff
@@ -390,11 +390,9 @@ export class SummonsData extends foundry.abstract.DataModel {
       ui.notifications.warn("DND5E.Summoning.Warning.Wildcard", { localize: true });
     }
 
+    actorUpdates["flags.dnd5e.summon.origin"] = this.item.uuid;
```

Comment:
Is this used anywhere? Do we need to update those references?

Reply:
We haven't referenced them yet, intended for a potential future feature but probably works better on the synthetic actor anyway.